tmt/convert.py

```diff
@@ -532,6 +541,9 @@ def read_nitrate_case(testcase, makefile_data=None):
         else:
             echo(style('environment:', fg='green'))
             echo(pprint.pformat(data['environment']))
+    # Possible multihost tag (detected in Makefile)
+    if makefile_data:
+        data['tag'] = makefile_data.get('tag', [])
```

Comment:
isn't possible that this part overwrite already stored `data['tag']` isn't better to use `.append()` here in case tag multihost tag exists? It is possible that that is fist usage of data['tag'] but also here I would love to use append, n future, when someone reorder priorities, may lead to strange outputs.